templates/zerver/emails/followup_day1.html

```diff
@@ -1,31 +1,30 @@
 {% extends "zerver/emails/email_base.html" %}
 
 {% block content %}
-    <p>Hi there!</p>
+<p>Hi there,</p>
 
-    <p>Welcome to Zulip.</p>
+<p>Welcome to Zulip!</p>
 
-    <p><a href="{{ server_uri }}/hello" style="color:#08c">{{ server_uri }}/hello</a> has a nice overview of what we're up to, but here are a few tips that'll help you get the most out of it:</p>
-
-    <ol>
-        <li>Zulip works best when it's always open, so <a href="{{ realm_uri }}/apps" style="color:#08c">grab our apps</a>! (Mac, Windows, Linux, Android, iOS)</li>
-
-        <li>Want to <b>share files</b> in Zulip? No problem! Start a new message and click the paperclip, or just drag-and-drop the file into the textbox.</li>
-
-        <li>Trying to <b>get someone's attention</b>? Type in an @-sign followed by their name to "mention" them. (Want more or fewer notifications? Select Settings under the gear menu to customize how and when Zulip notifies you of new messages.)</li>
-
-        <li>Bulleted lists, bold, code blocks, and more are all at your fingertips - just check out <b>"Message formatting"</b> in the gear menu.</li>
+<p>Zulip is the team communication solution that combines the real-time ease of chat with the threaded organization of email. At its core, Zulip is about productivity&mdash;making communication fun and easy, while avoiding the distracting and disorganized conversations of chatrooms. As an open-source project, we’re also all about building community, and we're happy that you’re here.
```

Comment:
Could you line-wrap this to 80 characters?
Also, I forget what the style is, but I think it might be 
```
<p>
lorem ipsum ..
</p>
```
or 
```
<p>
    lorem ipsum
</p>
```
Rather than starting the text on the same line as the `<p>`, in the cases where the text is more than 1 line.